packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -443,6 +444,10 @@ export default {
 
         updateMessage:  ({ commit }, message = null) => {
             commit(UPDATE_MESSAGE, message);
+        },
+
+        updateAddressFromLocalStorage: ({ commit }, address) => {
```

Comment:
I'd rename this to `updateAddress`. The content of the function doesn't indicate it's coming from local storage. In the future, we can reuse this function if the data is coming from other places.

Reply:
Fixed.